eslint.config.js

```diff
@@ -22,7 +22,41 @@ export default antfu({
     rules: baseRules,
 }, {
     files: ['**/*.ts', '**/*.tsx'],
-    ignores: ['**/*.d.ts', '**/vite.config.ts', 'playwright.config.ts'],
+    ignores: ['**/*.d.ts', '**/vite.config.ts', 'playwright.config.ts', '**/*.spec.ts', '**/*.spec.tsx`'],
+    plugins: {
+        header,
+    },
+    rules: {
+        'header/header': [
```

Comment:
It is not necessary to define the header rule twice in separate configurations, as it can be defined once in the main configuration.

Reply:
OK. Learned something new!